packages/headless/src/controllers/smart-snippet-questions-list/headless-smart-snippet-questions-list.ts

```diff
@@ -105,19 +120,12 @@ export function buildSmartSnippetQuestionsList(
     );
   };
 
-  const getIsExpanded = (
-    questionAndAnswer: QuestionAnswer,
-    relatedQuestions: QuestionAnsweringRelatedQuestionState[]
-  ) => {
-    const idx = findRelatedQuestionIdx(
```

Comment:
I'm not sure why we were looping through all of them, I would expect the related question to be at the same index in both the search response state and the question-answering state.